pkg/controllers/rollout/controller.go

```diff
@@ -563,6 +562,16 @@ func (r *Reconciler) calculateRealTarget(crp *fleetv1beta1.ClusterResourcePlacem
 // A binding with not trackable resources is considered ready if the binding's current spec has been available before
 // the ready cutoff time.
 func isBindingReady(binding *fleetv1beta1.ClusterResourceBinding, readyTimeCutOff time.Time) (time.Duration, bool) {
+	// the binding is ready if the diff report has been reported
+	diffReportCondition := binding.GetCondition(string(fleetv1beta1.ResourceBindingDiffReported))
```

Comment:
Another thing that I might not have fully thought through is that if we treat `DiffReported` bindings to be not-failed, ready bindings  it would be subject to rolling update strategy as well, which means it could also get stuck (e.g., member clusters get disconnected). Would this be a very confusing situation for end users (i.e., all the diffs reported would be stale)?